test/scripts/backup-restore/j07-verify-rhsso-backup-and-restore.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/sh
+# USAGE
+# ./j07-verify-rhsso-backup-and-restore <optional NS_PREFIX> <optional AWS_ACCESS_KEY_ID> <optional AWS_SECRET_ACCESS_KEY>
+# ^C to break
+# Tests RHSSO postgres back up and restore
+#
+# AWS_SECRET_ACCESS_KEY and AWS_ACCESS_KEY_ID should be set before running the script to support STS clusters as AWS
+# AWS credentials are not available on STS clusters. By default the script will try to use the credentials on cluster.
+#
+# PREREQUISITES
+# - oc (logged in at the cmd line)
 
 # Import the test function
 . ./postgres.sh --source-only
 
 # Set the parameters
+# Use passed in AWS credentials for STS, otherwise default to getting from cluster for normal OSD. Exit if these credentials are empty
+export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:=$(oc get secret aws-creds -n kube-system -o jsonpath='{.data.aws_access_key_id}' | base64 --decode)}
```

Comment:
same here

Reply:
When running this script `AWS_SECRET_ACCESS_KEY` and `AWS_ACCESS_KEY_ID` should be set before running the script to support STS clusters as credentials are not available on STS clusters. By default the script will try to use the credentials on cluster in other scenarios.